### ISSUE TITLE
view own posts on mobile app?

### ISSUE THREAD

**Issue by thequbit** [hhroc/yellr#57]
Should a user be able to view their own posts on the mobile app?  I'm thinking that might be useful to go back through and look at what you posted.  Thoughts @aztec8 ?

**Comment by Z4P0** [comment]
Yeah I can see how that'd be nice to have. I'm definitely leaning for it

**Comment by thequbit** [comment]
Okay.  I'll make a API that you can view posts by client_id.  This will be useful for the moderator side of things anyway.

I already have this for testing, documented here:

https://github.com/hhroc/yellr/wiki/API-Documentation#get_postsjson

I'll just add two variables to the GET:

```
/get_posts.json?client_id=xxxxyyyyzzzz&page=0
```
- client_id
  - unique id
- page
  - the page number.  I'll set it to return some number per page (10 or 20 or something), and that way we don't return too many at a time.

Make sense?

**Comment by Z4P0** [comment]
Sounds good

**Comment by thequbit** [comment]
This feature has been implemented in get_profile.json.

https://github.com/hhroc/yellr/wiki/API-Documentation#get_profilejson

Would need to be added to the app.
